src/graphics/systems/mesh.rs

```diff
@@ -236,7 +236,7 @@ pub fn create() -> Box<dyn Schedulable> {
                 command_buffer_queue
                     .push(CommandQueueItem {
                         buffer: encoder.finish(),
-                        name: "pbr".to_string(),
+                        name: "mesh".to_string(),
```

Comment:
Hmm, unfortunately as much as I like this fix here, it does have to be `pbr` if the order of dependencies is to be observed correctly. You likely don't see any issues, but in some cases the global buffers get updated later meaning the mesh will render with the previous frames lighting/transform information. A proper fix might be to introduce the concept of pipeline groups where a group of nodes is dependent on another group of pipelines. We could then have a "mesh" pipeline group which consists of `["pbr", "unlit"]` and depends on `["globals", "skybox"]`. Since that is a rather big change feel free to change this back to `pbr` or make the changes suggested above. 😃